kombu/utils/json.py

```diff
@@ -51,7 +53,7 @@ def default(self, o,
                 r = o.isoformat()
```

Comment:
```suggestion
                r = o.isoformat(timespec='microseconds')
```

Let's always serialize at the highest resolution Python provides us with.